README.md

```diff
@@ -55,7 +55,10 @@ TODO: Need update!
 
 ## QuickStart
 TBD
```

Comment:
Perhaps remove the TBD?

Reply:
I think we still have significant work to do here, but do not really care about this line. Does not have to be changed in this PR, IMHO.